lib/rubocop/cop/lint/redundant_safe_navigation.rb

```diff
@@ -4,8 +4,8 @@ module RuboCop
   module Cop
     module Lint
       # Checks for redundant safe navigation calls.
-      # Use cases where a constant is `nil` are rare and an offense is detected
-      # when the receiver is a constant.
+      # Use cases where a constant, named in camel case for classes and modules is `nil` are rare,
+      # and an offense is detected when the receiver is a snake case constant.
```

Comment:
I think you meant to write "is not detected" here.

Reply:
Ah, you mean to target just `SCREAMING_SNAKE_CASE`. I see.

Reply:
Oops! I've updated the text.